packages/batch-submitter/src/exec/run-batch-submitter.ts

```diff
@@ -4,8 +4,17 @@ import { Logger, Metrics } from '@eth-optimism/common-ts'
 import { exit } from 'process'
 import { Signer, Wallet } from 'ethers'
 import { JsonRpcProvider, TransactionReceipt } from '@ethersproject/providers'
-import { config } from 'dotenv'
-config()
+import * as dotenv from 'dotenv'
+import Config from 'bcfg'
+
+interface Bcfg {
```

Comment:
I think we have this in multiple places now. We could probably consider consolidating this into a single interface in `core-utils` or even make a PR against https://github.com/DefinitelyTyped/DefinitelyTyped adding typedefs for bcfg.

Reply:
I'm happy to move the interface into `core-utils` for now and fleshing out a full one against definitelytyped in the future